.github/workflows/scheduled-release.yml

```diff
@@ -36,6 +36,6 @@ jobs:
         with:
           source_branch: "release/${{ env.LATEST_RELEASE }}"
           destination_branch: "master"
-          pr_reviewer: "lgarofalo,maxnystrom,mihirjham,mitalirawat,nickysemenza"
```

Comment:
can this reference a group like `@cloudflare/ssl-tls` ? https://github.com/orgs/cloudflare/teams/ssl-tls/